packages/rest-hooks/tests/restClient.test.ts

```diff
@@ -102,15 +102,11 @@ describe('Test Rest Client', () => {
     defaultBaseUrl: 'http://fakehost.com:5023',
     timeout: 1000
   }
+  
   // used accross multiple tests, so declared here
   let rsp: StcRestTest.TestResponse
-
-  // TODO: only uncomment when needed, once all tests are converted, remove any not used
-  // let requestInfo: StcRestTest.RequestInfo
-  // let responseBody: StcRestTest.ResponseBody
-  // let data: any
-  // let restParams: StcRest.RestParams
-
+  let body: any
```

Comment:
@SteveAtSentosa should we call this `bodyData` instead?

Reply:
body and data in rest world are synonymous, but I see your point about consistency.  I think I prefer using `data` in all cases.  I am going to go ahead and merge this, because I need the tests, but we can certainly address this in a future MR ... consistency is good :)

Reply:
Sounds good, @SteveAtSentosa I agree, just minor details we can clean later.